src/js/components/PodInstancesView.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
Let's make sure to move this under `pages/` in the folder structure, when we get these tabs routed. No change needed now